jiant/tasks/lib/qamr.py

```diff
@@ -0,0 +1,73 @@
+import pandas as pd
+from dataclasses import dataclass
+
+from jiant.tasks.lib.templates import span_prediction as span_pred_template
+from jiant.utils.python.io import read_json_lines
+from jiant.utils.retokenize import TokenAligner, MosesTokenizer
+
+
+class QAMRTask(span_pred_template.AbstractSpanPredicationTask):
+    def get_train_examples(self):
+        return self._create_examples(self.train_path, set_type="train")
+
+    def get_val_examples(self):
+        return self._create_examples(self.val_path, set_type="val")
+
+    def get_test_examples(self):
+        return self._create_examples(self.test_path, set_type="test")
+
+    def _create_examples(self, qa_file_path, set_type):
+        wiki_df = pd.read_csv(self.path_dict["wiki_dict"], sep="\t", names=["sent_id", "text"])
+        wiki_dict = {row["sent_id"]: row["text"] for _, row in wiki_df.iterrows()}
+
+        data_df = pd.read_csv(
+            qa_file_path,
+            sep="\t",
+            header=None,
+            names=[
+                "sent_id",
+                "target_ids",
+                "worker_id",
+                "qa_index",
+                "qa_word",
+                "question",
+                "answer",
+                "response1",
+                "response2",
+            ],
+        )
+        data_df["sent"] = data_df["sent_id"].apply(wiki_dict.get)
+
+        examples = []
+        moses_tokenizer = MosesTokenizer()
+        for i, row in data_df.iterrows():
+            # Answer indices are a space-limited list of numbers.
+            # We simply take the min/max of the indices
+            answer_idxs = list(map(int, row["answer"].split()))
```

Comment:
I wrote a very basic test to see your code run (all it does so far is load data like this part of the [test for sst](https://github.com/jiant-dev/jiant/blob/59ac46b7812bfe88277a153a250b2db5b12ec993/tests/tasks/lib/test_sst.py#L94-L98)).

The test fails at this line because `row["answer"].split()` assumes `row["answer"]` is a string, but when you `iterrows()` over the dataframe for some rows the value's type may be inferred as numeric.

Side note: this data may not be very big, so it may not make sense to optimize, but fyi iterrows [is not very efficient](https://stackoverflow.com/a/59413206/8734015).